assets/src/edit-story/components/canvas/karma/rtl.karma.js

```diff
@@ -45,9 +50,10 @@ describe('RTL support', () => {
   describe('CUJ: Creator can Select an Element: Transforming element', () => {
     it('should allow dragging element in RTL mode', async () => {
       await fixture.events.click(fixture.editor.library.textAdd);
+      await waitFor(() => fixture.editor.canvas.framesLayer.frames[1].node);
```

Comment:
The text will now take more time for adding than before, especially in case of tests since there will be not enough time to generate the image before clicking. Wondering if there's a better way to consider this in the tests...